netcon/onlinedirectipclient/chat_api.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Descent 3
+ * Copyright (C) 2024 Parallax Software
```

Comment:
Since this is new code, please change copyright to `Copyright (C) 2024 Descent Developers` here and on all new files.